docs/adding_icons.md

```diff
@@ -0,0 +1,29 @@
+If you would like to add an icon, the easiest way to do this is to contact Axa or Shaun. The process involves splitting the SVG markup of the iconset that we use, to create individual icon components that are dynamically filled, sized, and labeled.
```

Comment:
@ishasinha1 can you remove the first sentence and rephrase this to:  
`The process of adding an icon involves splitting the SVG markup of the iconset that we use, to create individual icon components that are dynamically filled, sized, and labeled.`